app/assets/stylesheets/components/_avatar.scss

```diff
@@ -18,3 +18,10 @@
   box-shadow: 0 1px 2px rgba(0,0,0,0.2);
   border: white 1px solid;
 }
+
+.avatar-huge {
+min-width: 300px;
+height: 300px;
```

Comment:
Could you add some indentation so that the code is cleaner?

Also there is an empty line at the end of this. Will probably touch the code again in the end but best to start off clean I guess :)

Reply:
fixed, check pr #14